subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r31/PersistentCompositeDependencySubstitutionCrossVersionSpec.groovy

```diff
@@ -102,7 +104,10 @@ class PersistentCompositeDependencySubstitutionCrossVersionSpec extends ToolingA
         ideaModule.dependencies.size() == 1
         with(ideaModule.dependencies.first()) {
             it instanceof IdeaModuleDependency
-            dependencyModule == null
+            // this currently fails for current -> 3.5
+            if (targetVersion < GradleVersion.version("4.0")) {
```

Comment:
You can use `GradleVersion.current()`. It might be good to add a helper method that does this on one of the fixtures to make this more explicit.